README.md

```diff
@@ -7,144 +7,119 @@
-echo 'export PATH="$GEM_HOME/bin:$PATH"' >> ~/.zshrc # Needed if any gems have executables associated.
-source ~/.zshrc # Let's pick up those changes.
-gem install bundler # Bundler is the standard for dependency management. Install it system-wide.
+``` shell
+./bin/configure
 ```
 
-Afterwards, clone this repository and install dependencies:
+This will install the following:
 
-```shell
-nvm use system # If it ain't broke - we might pin this in the future.
-npm install
+-   [asdf: a version manager used for other dependencies](https://asdf-vm.com/)
+    note: it will *not* be added to your `$PATH`, but used by the scripts in
+    this repo
+    -   asdf will install (defined:in [`.tool-versions`](.tool-versions))
+        -   node
+        -   python
+        -   ruby
+-   [maven: java package manager for swagger](https://maven.apache.org/)
+    -   [swagger](https://github.com/swagger-api/swagger-codegen)
 
-rbenv shell system # We'll use system Ruby as well, but just like Node, we might pin it in the future.
-bundle install
-```
-
-You also might need some python dependencies:
-```shell
-pip install ruamel.yaml
-```
+## Local Development
 
-For the rest of this README, assume any commands that we list will be run from the root of this repository.
+To build once:
 
-## How to build and run documentation locally
-
-To generate and view documentation on your local machine, follow the steps below:
-
-1. Navigate to the root directory of this project. Then, run the "first time setup" command (see other commands in the [Makefile](https://github.com/Asana/developer-docs/blob/master/Makefile)):
-
-```
-make first_time_setup
+``` shell
+make
 ```
 
-2. Update build tools and client libraries:
+To reset the working directory:
 
-```
-make update
+``` shell
+make clean
 ```
 
-3. Generate documentation (internal Asanas: see [this task](https://app.asana.com/0/0/1200652548580470/f) before running):
+To build continouously upon changes and launch the local web server:
 
-```
-make docs_gen
+``` shell
+./bin/watch_and_serve_local_dev
 ```
 
-4. Serve documentation locally:
+Documentation will be accessible here (also included in the build output):
 
-```
-make serve
-```
+-   <http://localhost:4567/>
 
-5. View documentation in your browser by visiting **http://localhost:4567/docs** (or **http://127.0.0.1:4567/docs**).
+## Making Changes to Changes
 
-## Reference (how it works)
-The public OpenAPI spec is located at **/defs/asana_oas.yaml**
+*Internal Asanas: See <https://app.asana.com/0/77076599077/1122503737028047/f>
+and <https://app.asana.com/0/0/1200652548580470/f> before making any updates.*
 
-To generate markdown from the spec, we use a forked [widdershins](https://github.com/rossgrambo/widdershins).
+If the content you're changing is static (not generated from the OpenAPI spec),
+you'll edit the `.md` files in source/includes/markdown.
```

Comment:
Might also be worth a mention: If you are creating an entirely new markdown file (rather than editing an _existing_ one), you will need to create the markdown file yourself, then reference it in [/source/docs/index.html.md](https://raw.githubusercontent.com/Asana/developer-docs/master/source/docs/index.html.md)